### PR TITLE
Add cinder husvm config backport to cloud6

### PR DESCRIPTION
**Pull request by Itxaka**
cherry-picked from https://github.com/crowbar/crowbar-openstack/pull/464/commits/63fbcbfd966679b3edd5214b2afad0f0ff31a072

Adapted the migration-schema to conform to cloud6 naming (Will squash commits later)

### CODE REVIEW

**Comment by Itxaka** [comment]
Updated the backport with latest changes.

File: chef/cookbooks/cinder/templates/default/cinder.conf.erb [hunk 1]
```diff
@@ -2385,6 +2385,13 @@ volume_driver = cinder.volume.drivers.zfssaiscsi.ZFSSAISCSIDriver
 <% if volume['backend_driver'] == 'glusterfs' -%>
 volume_driver = cinder.volume.drivers.glusterfs.GlusterfsDriver
 <% end -%>
+<% if volume['backend_driver'] == 'hitachi' -%>
```
Comment (aplanas) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb]: can we use `elsif` to make it clear that you are addressing the same configuration key?

File: chef/cookbooks/cinder/templates/default/cinder.conf.erb [hunk 2]
```diff
@@ -2385,6 +2385,13 @@ volume_driver = cinder.volume.drivers.zfssaiscsi.ZFSSAISCSIDriver
...
+  <% if volume['hitachi']['hitachi_storage_protocol'] == 'iscsi' -%>
+volume_driver = cinder.volume.drivers.hitachi.hbsd_iscsi.HBSDISCSIDriver
+  <% elsif volume['hitachi']['hitachi_storage_protocol'] == 'fc' -%>
+volume_driver = cinder.volume.drivers.hitachi.hbsd_fc.HBSDFCDriver
+  <% end -%>
+<% end -%>
 <% if volume['backend_driver'] == 'manual' -%>
```
Comment (aplanas) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb]: same here : ?

**Comment by aplanas** [comment]
LGTM +1

File: chef/cookbooks/cinder/templates/default/cinder.conf.erb (hunk 1 above)
Comment (Itxaka) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb]: Well...for once it keeps with the style of other lines (all of them use if) and also I dont feel confortable changing only a backport, should be fixed on master then backported instead.

**Comment by rsalevsky** [comment]
LGTM feel free to merge

**Pull request merged**